crates/load-test/src/main.rs

```diff
@@ -0,0 +1,450 @@
+//! Load test for pathfinder JSON-RPC endpoints.
```

Comment:
Consider adding this to the `[package]` as the description.

Reply:
Done.